draft-duke-quic-load-balancers.md

```diff
@@ -360,17 +369,16 @@ normative:
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                        Version (32)                           |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|      0x00     |
-+-+-+-+-+-+-+-+-+
+|  0x00 | 0x00  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                  Authentication Token (64)                    +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|      CR       | Message Type  |
```

Comment:
CR is only 2 bits. I took these two bits out of the message type, but it'd be perfectly fine to have a whole byte with 6 reserved bits, if you prefer.